packages/flow-core/src/components/f-accordion/f-accordion.scss

```diff
@@ -13,8 +13,15 @@
 			transform: rotate(180deg);
 			transition: transform var(--transition-time-default) linear;
 		}
+
+		&.doc-preview-accordion {
```

Comment:
We must try `part` feature

https://developer.mozilla.org/en-US/docs/Web/CSS/::part